test/test_optim.py

```diff
@@ -431,6 +431,21 @@ def test_adam(self):
             with self.assertRaisesRegex(ValueError, "Invalid weight_decay value: -1"):
                 optimizer(None, lr=1e-2, weight_decay=-1)
 
+    # Test whether variance parameter is always real
+    def test_complex_adam_variance(self):
```

Comment:
How long does this test takes? Should it be marked as slow?

Reply:
Currently this test takes 0.5 second, moreover, actually having 200 iterations is unnecessary, so I am reducing it to 20 iterations, which makes runtime less than 0.1 second. I believe this is not qualified as to be slow, right ?

Reply:
Definitely not